packages/apify/src/actor.ts

```diff
@@ -1352,6 +1351,40 @@ export interface MetamorphOptions {
 export interface ExitOptions {
     /** Exit code, defaults to 0 */
     exitCode?: number;
+    /** Call `process.exit()`? Defaults to true */
+    exit?: boolean;
+}
+
+export interface OpenStorageOptions {
+    /**
+     * If set to `true` then the cloud storage is used even if the `APIFY_LOCAL_STORAGE_DIR`
+     * environment variable is set. This way it is possible to combine local and cloud storage.
+     * @default false
+     */
+    forceCloud?: boolean;
+}
+
+/**
+ * Represents information about an actor run, as returned by the {@link Actor.call} or {@link Actor.callTask} function.
+ * The object is almost equivalent to the JSON response of the [Actor run](https://apify.com/docs/api/v2#/reference/actors/run-collection/run-actor)
+ * Apify API endpoint and extended with certain fields. For more details, see [Runs.](https://docs.apify.com/actor/run)
+ */
+export interface ActorRunWithOutput extends ClientActorRun {
+    /**
+     * Contains output of the actor run. The value is `null` or `undefined` in case the actor is still running,
+     * or if you pass `false` to the `fetchOutput` option of {@link Actor.call}.
+     *
+     * For example:
+     * ```
+     * {
+     *   "contentType": "application/json; charset=utf-8",
+     *   "body": {
+     *     "message": "Hello world!"
+     *   }
+     * }
+     * ```
+     */
+    output?: Dictionary | null;
```

Comment:
I would keep the `call` API exactly the same as Client now. The final version of how this whole output management will look like is still undecided. So there should be no need for this type.

Reply:
This PR is not about that, the diff is just about moving types from a different package to the actor sdk.

But good catch, this is indeed forgotten type that is wrong (there is no `fetchOutput` anymore in the code), will remove it.

Reply:
Not sure I understand. You can't have docs for a method you don't have a separate interface for?

Reply:
Sure we can, but the client type has no jsdoc at all for `ActorRun`, and it does not make much sense to have jsdoc talking about the `Actor` class from SDK in client.

Reply:
I think having the return value documented in the method will be enough.